extensions/binding/org.eclipse.smarthome.binding.sonyaudio/ESH-INF/thing/ht-ct800.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sonyaudio"
+							xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+							xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+							xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Commands -->
+	<channel-type id="input-ct800">
+		<item-type>String</item-type>
+		<label>Input Source</label>
+		<description>Select the input source of the reciever</description>
+		<state>
+				<options>
+						<option value="extInput:btAudio">Bluetooth Audio</option>
```

Comment:
Yes it would be nicer, but I did it in this way for flexibility, the "extInput:hdmi?port=3" is the command sent to the device this can be different for different models for example the ZR5 needs "extInput:line?port=1" but the CT800 needs "extInput:line" without "port" to set "Analog" input.
For the moment the rest of the code don't know anything about the specific device, this is in the "thing" xml.
The alternative is to move this knowledge to the handler?

Reply:
From a user perspective, it would be nicer if the different "Things" look similar wrt the options and that technical differences are indeed handled internally in the binding.

So yes, I think it would make sense to have some mapping tables in the handler, where you can get the technical string for an input source of a given thing type.